tests/integration/utils.rs

```diff
@@ -1,6 +1,7 @@
 
 use fred;
 
+use rand::distributions::Alphanumeric;
```

Comment:
Tests weren't compiling since they didn't have the same changes made in https://github.com/azuqua/fred.rs/pull/92/commits/1aecf6b4e5bb9013fc244950db31dd626f018267.